src/app/services/info/info.service.ts

```diff
@@ -58,4 +61,19 @@ export class InfoService {
       );
   }
 
+  private getSegregatedData = (category: string, id: string): Observable<object> => {
```

Comment:
A better name will be `getFilteredData` since we are just filtering:
```suggestion
  private getFilteredData = (category: string, id: string): Observable<object> => {
```